akka-docs/src/main/paradox/typed/replicated-eventsourcing-examples.md

```diff
@@ -5,9 +5,16 @@ The following are more realistic examples of building systems with Replicated Ev
 @@toc { depth=1 }
 
 @@@ index
-
 * [auction](replicated-eventsourcing-auction.md)
 * [shopping cart](replicated-eventsourcing-cart.md)
-
 @@@
 
+## Additional samples
```

Comment:
Couldn't get the external links in the index/toc but this header ends up in the index leading to this page. Perhaps not great, but it works.